source/_integrations/yale.markdown

```diff
@@ -0,0 +1,156 @@
+---
+title: Yale
+description: Instructions on how to integrate your Yale devices into Home Assistant.
+ha_category:
+  - Binary sensor
+  - Button
+  - Camera
+  - Doorbell
+  - Event
+  - Lock
+  - Sensor
+ha_release: 2024.9
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@bdraco'
+ha_domain: yale
+ha_dhcp: true
+ha_platforms:
+  - binary_sensor
+  - button
+  - camera
+  - diagnostics
+  - event
+  - lock
+  - sensor
+ha_integration_type: integration
+---
+
+The **Yale** integration allows you to integrate your [Yale](https://www.assaabloy.com/vn/en/solutions/products/yale) devices in Home Assistant.
+
+The login credentials used to authenticate the integration are the same as the ones used in the Yale Home app.
+
+{% include integrations/config_flow.md %}
+
+## Known working devices
+
+| Device                            | Requires [Connect Bridge](https://www.yalehome.com/vn/en/products/yale-smart-door-lock/smart-accessories/yale-connect-wi-fi-bridge) or Doorbell |
+| --------------------------------- | ------------------------------------|
+| Yale Assure Lock | yes |
+| Yale Conexis L1 | yes |
+| Yale Conexis L2 | yes |
+| Yale Doorman L3 | yes |
+| Yale Linus | yes |
+| Yale Smart Safe | yes |
```

Comment:
All the Yale Unity products probably work here as well but we didn't have them listed on either and I don't live in the AU market or have access to these devices